QModManager/ErrorDialog.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace QModManager
+{
+    internal static class Dialog
+    {
+        internal static void Show(string error, Action onLeftButtonPressed = null, Action onRightButtonPressed = null, string leftButtonText = "See Log", string rightButtonText = "Close", bool blue = false)
```

Comment:
Maybe have the text parameters before the Action parameters.

Reply:
I don't think that this is necessary as there will always be `Action` parameters when there are the `string` parameters. Also, again, most of them won't be used in the final build. And if they will, they'll be named.

Reply:
Guess it makes sense.